content/en/blog/2022-01-31-january-update/index.md

```diff
@@ -0,0 +1,538 @@
+you will notice that almost all the tasks have been done (or are indeed
+very close). It's largely the fuzzing work which is close to land and
+some additional documentation. 2.5 months after the audit concluded we
+are quite happy with where we have arrived - having the analysis of the
+auditors in front of us gave us a solid focus.
+
+In the 0.26 release of Flux, we applied the [restricted pod security
+standard](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted)
+to controllers like [source-controller](https://github.com/fluxcd/source-controller/pull/553).
+In practice this will mean:
+
+- all Linux capabilities were dropped
+- the root filesystem was set to read-only
+- the `seccomp` profile was set to the runtime default
+- run as non-root was enabled
+- the filesystem group was set to 1337
+- the user and group ID was set to 65534
+
+Flux will also enable `seccomp` by default across all controllers. Why is
+this important? Well, the default `seccomp` profile blocks key system
+calls that can be used maliciously, for example to break out of the
+container isolation. The recently disclosed [kernel vulnerability
+CVE-2022-0185](https://blog.aquasec.com/cve-2022-0185-linux-kernel-container-escape-in-kubernetes)
+is a good example of that.
+
+Big news are also that we
+
+- [Publish SBOM for Flux and the GitOps Toolkit
+  components](https://github.com/fluxcd/flux2/issues/2302)
+  and
+- [Sign the release checksums and container images with Cosign and
+  GitHub
+  OIDC](https://github.com/fluxcd/flux2/issues/2303)
+
+One bit we are still working on and will be part of the next release is
+upgrading our [dependency libgit2 to
+1.3.0](https://github.com/fluxcd/source-controller/pull/557),
+which will add support for ed25519 for both client authentication and
+hostKey verification.
+
+### A word on RFCs
+
+After reviews from all maintainers, [RFC-0001
+Authorization](https://github.com/fluxcd/flux2/tree/main/rfcs/0001-authorization)
+is merged. This RFC describes in detail, for [Flux version
+0.24](https://github.com/fluxcd/flux2/releases/tag/v0.24.0),
+how Flux determines which operations are allowed to proceed, and how
+this interacts with Kubernetes\' access control.
+
+To this point, the Flux project has provided [examples of how to make a
+multi-tenant
+system](https://github.com/fluxcd/flux2-multi-tenancy/tree/v0.1.0),
+but not explained exactly how they relate to Flux\'s authorization
+model; nor has the authorization model itself been documented. Further
+work on support for multi-tenancy, among other things, requires a full
+account of Flux\'s authorization model as a baseline.
+
+**Goals**: Give a comprehensive account of Flux\'s authorization model
+
+**Non-Goals**: Justify the model as it stands; this RFC simply records
+the state as at v0.24.
+
+#### Flux in-flight RFCs
+
+[https://github.com/fluxcd/flux2/labels/area%2FRFC](https://github.com/fluxcd/flux2/labels/area%2FRFC)
+
+XXX - help...??
```

Comment:
@fluxcd/flux2-maintainers what would you like me to say about the in-flight RFCs?

Reply:
RFCs are out of date as they don't reflect the changes in flux2 0.26. I don't have time to update all the RFC today, so we either postpone the blog post until I fix them or we leave them out.

Reply:
Maybe let's do them separately - it might be a good idea to do a separate blog post and list to the mailing list about this...?

Reply:
yeap

Reply:
super... :+1: